### ISSUE TITLE
Groups

### ISSUE THREAD

**Issue by evanp** [pump-io/pump.io#299]
We need to support groups. I think a group should just repeat objects that are posted to it, bcc all group members. Is there more to it than that?

**Comment by salyavin** [comment]
I agree with vasygan.  Groups are a must. Groups make it easier to have conversations with those you do not follow about a particular topic.  It is one of th things that made status.net great.  Without it I would not expect pump.io to be used much.

**Comment by vasyugan** [comment]
I guess the broader problem is: Pump.io development is dead, no commit for half a year or so. R.I.P.

**Comment by evanp** [comment]
Ping!
